src/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.RunAsClient.Windows.cs

```diff
@@ -13,8 +13,9 @@ namespace System.IO.Pipes.Tests
 {
     public partial class NamedPipeTest_RunAsClient : RemoteExecutorTestBase
     {
-        public static bool IsNotWinRTOrIsWindows10Version16256OrGreater => PlatformDetection.IsNotWinRTSupported || PlatformDetection.IsWindows10Version16256OrGreater;
+        public static bool IsNotWinRTOrIsWindows10Version16256OrGreater => PlatformDetection.IsNotInAppContainer || PlatformDetection.IsWindows10Version16256OrGreater;
 
+        [ActiveIssue("https://github.com/dotnet/corefx/issues/24318")]
```

Comment:
Looks like this is only failing on uapaot, should we instead just skip it there instead of every run?

Reply:
Let me check uap also, but, yes, it makes more sense to skip only on the affected ones.

Reply:
oh - I missed that - you're right @joperezr